docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,98 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant additional two years of 
+extend security coverage. This service is only available on Trusty, since the release has reached
+its end of life support for esm-infra.
+
+.. Make sure Pro is up to date
+.. include:: ./enable-disable/update-pro.txt
+
+Check the status of the services
+================================
+
+After you have :ref:`attached your subscription <get_token_and_attach>` and
```

Comment:
for similar reasons as above, I don't think we can link to that article here. It will likely include features that don't exist in trusty (even if it doesn't now, it may in the future)